### DISCUSSION TITLE
how to import a single song from an album

### DISCUSSION THREAD

**Discussion by jrunosss** [beetbox/beets#5451] [Q&A]
i cant figure out how to import a song from an album without importing it as a singleton. for example, if there is an album called X and i have track 03 from that album, how do i import it as a track in that album instead of being a singleton.

**Comment by arogl** [comment]
If you only have the 1 track in the album and beets identifies the album correctly, you can still import and it will be tagged with all the right metadata. Just choose the album from the list presented.

If the album is not tagged or identified correctly, you can find the release and pass it through like `beet imp --search-id SEARCH_ID` where `SEARCH_ID` can be the full URL of the album, or just the ID 

Just make sure you make don't use the `-q` option when importing.
